e2e/cypress/integration/integrations/incoming_webhook/cancel_out_of_edit_spec.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @integrations
+
+describe('Integrations', () => {
+    before(() => {
+        // # Create test team, channel, and webhook
+        cy.apiInitSetup().then(({team, channel}) => {
+            const newIncomingHook = {
```

Comment:
Please move up `newIncomingHook` so we can use it in the `it` block, instead of verifying against hard-coded values.
```suggestion
    let newIncomingHook;

    before(() => {
        // # Create test team, channel, and webhook
        cy.apiInitSetup().then(({team, channel}) => {
            newIncomingHook = {
```

Reply:
Yup makes sense.